app/controllers/sessions/manage_consent_reminders_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Sessions::ManageConsentRemindersController < ApplicationController
+  before_action :set_session
+
+  def show
+    authorize :consent_reminder, :show?
```

Comment:
```suggestion
    authorize :consent_reminder
```

You shouldn't need to specify the action name as it'll take it automatically.